scripts/CMSRunAnalysis.py

```diff
@@ -742,7 +742,7 @@ def StripReport(report):
         print("== Execution site from site-local-config.xml: %s" % slCfg.siteName)
         with open('jobReport.json', 'w') as of:
             json.dump(rep, of)
-        with open('jobReportExtract.pickle', 'w') as of:
+        with open('jobReportExtract.pickle', 'wb') as of:
```

Comment:
is this py2 safe ? CMSRunAnalysis.py is executed in the job wrapper in the WN. We currently run the wrapper with python2

Reply:
I am 99% sure this will work in py2. 

- WMCore is making extensive use of opening pickle files with "wb": https://github.com/dmwm/WMCore/search?p=1&q=%22wb%22
- https://github.com/dmwm/WMCore/pull/10742 many of those have been modified in this PR.

The missing 1% come from the fact that I did not try it myself right now, just based this answer on my memory, and memory can fail me. In any case, it seems very unlikely it does in this very case!

Reply:
sure, let's build and test the new TW